articlequality/utilities/fetch_text.py

```diff
@@ -74,7 +74,8 @@ def fetch_text(session, labelings, verbose=False):
     """
 
     for labeling in labelings:
-        rev_doc = get_last_rev_before(session, labeling['page_title'],
+        subject_page_id = get_subject_page_id(session, labeling['talk_page_id'])
+        rev_doc = get_last_rev_before(session, subject_page_id,
                                       labeling['timestamp'])
 
         if rev_doc is None:
```

Comment:
Should it write the talk page id instead?

Reply:
I'm not sure what you mean.

Reply:
Currently, there is the following code below:
```python
sys.stderr.write(labeling['page_title'] + " " + labeling['timestamp'])
```
Should it write the talk page name instead?

Reply:
Oh!  I see.  I think we want the subject_page_id.  This output is meant to let us know when a page could not be found.

Reply:
Oh!  We'd want to handle the case where we can't find the subject page ID.  That'd be more common.  I think outputting the talk_page_id makes sense then.